sphinx/parsers.py

```diff
@@ -55,15 +53,16 @@ def set_application(self, app):
         self.info = app.info
 
 
-class RSTParser(docutils.parsers.rst.Parser):
+class RSTParser(docutils.parsers.rst.Parser, Parser):
```

Comment:
`Parser` super class is no longer needed.

Reply:
done